src/asr/asr_chainer.py

```diff
@@ -244,7 +242,46 @@ def run(self):
                 scatter_params(self.model, gp)
                 gp = None
 
-                delete_feat(x)
+
+class CustomConverter(object):
+    """CUSTOM CONVERTER"""
+
+    def __init__(self, device, subsamping_factor=1):
+        self.device = device
+        self.subsamping_factor = subsamping_factor
+        self.xp = cuda.cupy if device != -1 else np
+
+    def __call__(self, batch, *args):
```

Comment:
Why `args` is required? It seems not to be used in this function and not to be given when this function is called anywhere.

Reply:
In `chainer.extensions.Eavaluator`, converter recieves 2 argments.
So I added but there may be more smart way :(

Reply:
> In addition, it is useful to prefetch if loading from scp file becomes other function.
Is it necessary? I think just replacing converter is OK for your prefetcher.

Reply:
I see. Then it is better to send data to GPU in `converter` with `device` argument. It fits to Chainer's manner.

Reply:
If `device` argument and loading data to GPU is supported, `CustomWorker` and `setup_workers` may be removed.

Reply:
I will improve interface to be compatible chainer manner.